android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java

```diff
@@ -51,6 +53,17 @@
         EXCLUDE_SESSIONS.add("http://it.droidcon.com/2015/sessions/barcamp-organization-2/");
         EXCLUDE_SESSIONS.add("http://it.droidcon.com/2015/sessions/droidcon-party/");
         EXCLUDE_SESSIONS.add("http://it.droidcon.com/2015/sessions/chiusura/");
+
+        ROOMS_TO_COLORS = new HashMap<>();
```

Comment:
I could, but I didn't know earlier how many rooms we have... fixing it but... aren't we like compiling against Java 7? o.o